targets/TARGET_STM/TARGET_STM32L0/device/stm32l0xx_hal_conf.h

```diff
@@ -136,6 +136,9 @@
 #endif /* LSI_VALUE */                      /*!< Value of the Internal Low Speed oscillator in Hz
                                              The real value may vary depending on the variations
                                              in voltage and temperature.*/
+#if !defined  (LSI_ACCURACY)
```

Comment:
we should not be   touching these device files as they get updated from upstream. This could be defined in our codebase (config for instance), and used in the watchdog HAL

Reply:
@ARMmbed/team-st-mcd please review

Reply:
I agree, don't patch files in device directory (except bugs)
Thx